.changeset/weak-kings-behave.md

```diff
@@ -0,0 +1,6 @@
+---
+"@biomejs/backend-jsonrpc": minor
+"@biomejs/biome": minor
+---
+
+Added the `no-alert` rule from ESLint.
```

Comment:
``````suggestion
---
"@biomejs/biome": patch
---

Added a new nursery rule `noAlert` that disallows the use of `alert`, `confirm` and `prompt`.

The following code is deemed incorrect:

```js
alert("here!");
```
``````